src/navigation/navigation.scss

```diff
@@ -55,6 +55,14 @@ nav {
                font-size: 14px;
                cursor: pointer;
             }
+            .subgroup {
+               min-width: 130px;
+               background: #2A3052;
```

Comment:
use this variable $color-blue-2;

Reply:
@hirad-rewok unfortunately we don't have theme colors in WebTrader project.

Reply:
Yep, we don't have any color scheme in this project